pkgs/applications/science/machine-learning/weights/keras/common.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenvNoCC, fetchurl, model, version, weights, sha256 }:
+
+stdenvNoCC.mkDerivation rec {
+  inherit version;
+  name = "${model}-${version}";
+  src = fetchurl {
+    inherit sha256;
+    url = "https://github.com/keras-team/keras-applications/releases/download/${model}/${weights}";
+  };
+  installPhase = ''
+    ln -s "${src}" "$out"
+  '';
+  phases = [ "installPhase" ];
+  meta = with stdenvNoCC.lib; {
+    homepage = "https://github.com/keras-team/keras-applications";
+    description = "Keras ${name} weights";
```

Comment:
Funny. I wasn't sure of that but yes, it does work! Thanks, TIL.

Reply:
What were you not sure about?

Reply:
Using variable inside of `meta` to compose the description or the homepage